### PR TITLE
ME-1777: creating deb packages and signed repo in s3

### PR DESCRIPTION
**Pull request by th3wingman**
# Description

Adding deb packages creation with signed repo in s3 under downloads.border0.com/repos/

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

tested locally and on EC2
apt-g update, updrage and remove passes

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by adrianosela — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,7 +9,43 @@ Please check the full documentation here: [https://docs.border0.com/](https://do
...
+```
+echo "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/border0.gpg] https://download.border0.com/repos/ stable main" | sudo tee /etc/apt/sources.list.d/border0.list
+```
+
+Install the border0 package
+```
+sudo apt-get update
+sudo apt-get install border0
+```
+elternatively, we can specify the token as an environment variable:
```
Comment (adrianosela) [inline README.md:30]: typo

File: README.md [hunk 2]
```diff
@@ -9,7 +9,43 @@ Please check the full documentation here: [https://docs.border0.com/](https://do
...
+#!/bin/bash
+apt-get -y update
+apt-get -y install curl gnupg
+install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.border0.com/repos/gpg | gpg --dearmor -o /etc/apt/keyrings/border0.gpg
+echo "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/border0.gpg] https://download.border0.com/repos/ stable main" > /etc/apt/sources.list.d/border0.list
+apt-get -y update
+BORDER0_CONNECTOR_TOKEN=PUT_YOUR_CONNECTOR_TOKEN_HERE apt-get -y install border0 
+
+```
```
Comment (adrianosela) [inline README.md:46]: we should have an example on how to not verify the signature and warn against it -- but still provide it maybe?

File: DEBIAN/postinst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      # if token is set, run automated install in noninteractive mode
+      border0 connector install --v2 --daemon-only
+      create_config_file
+    else
+      echo -e "BORDER0_CONNECTOR_TOKEN is not set.\nPlease run the install manually... \n'border0 connector install --v2'"
+    fi
+  elif [ -n "$BORDER0_CONNECTOR_TOKEN" ]; then
+    # if token is set, run automated install in interactive mode
+    border0 connector install --v2 --daemon-only
+    create_config_file
```
Comment (adrianosela) [inline DEBIAN/postinst:30]: should it not restart the systemd unit / daemon?

**Review by adrianosela — APPROVED** [review]
Looks good, comments left.

File: DEBIAN/postinst (hunk 3 above)
Comment (th3wingman) [reply DEBIAN/postinst:30]: dpkg does it for us

File: README.md (hunk 2 above)
Comment (th3wingman) [reply README.md:46]: we can include that in further updates

**Pull request merged**